circle.yml

```diff
@@ -19,8 +19,7 @@ dependencies:
 
 test:
   override:
-    - echo "Skipped...Running our uitests on every pull request breaks"
-    - echo "them irregularly and they're badly maintained at the moment."
+    - echo "Beep Boop"
```

Comment:
Let's make this more useful. There are no tests to run because we're running them in Travis CI.